deploy/helm/hiphops/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hops
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: hops
+  template:
+    metadata:
+      namespace: {{ .Values.namespace }}
+      labels:
+        app: hops
+    spec:
+      serviceAccountName: hops-worker
+      containers:
+        - name: hops
+          image: "hiphops/hiphops:{{ .Values.hiphops.imageVersion }}"
+          command: ["/hops"]
+          args: ["start", "--address=0.0.0.0:8916", "--keyfile=/root/hiphops-key/hiphops.key", "--hops=/root/hops-conf/", "--serve-k8sapp"]
```

Comment:
@AceFire6 I'll make a new release in a moment which will include your non-root user changes, so then I think this can be updated to use those paths too.

Reply:
Awesome, I've already got those changes locally from when I was testing the new Dockerfile. I'll push those up later/tomorrow 👍